server/gitrest/src/externalStorageManager.ts

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import Axios from "axios";
+import * as winston from "winston";
+
+export interface IExternalStorageManager
+{
+    readAndSync(tenantId: string, documentId: string): Promise<boolean>;
+
+    writeFile(tenantId: string, ref: string, sha: string, update: boolean): Promise<void>;
+}
+
+/**
+ * Manages api calls to external storage
+ */
+export class ExternalStorageManager implements IExternalStorageManager {
+    constructor(private endpoint: string) {
+    }
+
+    public async readAndSync(tenantId: string, documentId: string): Promise<boolean> {
+        if (process.env.external_storage_enabled != "true") {
+            winston.info("External storage is not enabled");
+            return false;
+        }
+        winston.info(`Gitrest calling read from external storage on tenant ${tenantId} / ${documentId}`);
+        await Axios.post<void>(
+            `${this.endpoint}/file/${tenantId}/${documentId}`,
+            {
+                headers: {
+                    "Accept": "application/json",
+                    "Content-Type": "application/json",
+                },
+            }).catch((error) => {
+                winston.error(`Axios error ${error} tenantId ${tenantId}`);
+                throw error;
+            });
+
+        winston.info("Read and sync successful");
+        return true;
+    }
+
+    public async writeFile(tenantId: string, ref: string, sha: string, update: boolean): Promise<void> {
+        winston.info(`Gitrest calling write to external storage on tenant ${tenantId}`);
```

Comment:
The loggings were just for the POC. Remove any unnecessary logging like this one

Reply:
Can I keep them but make them verbose winston logs? I think having tenantId and documentId in the logs would be useful for debugging.

Reply:
Spoke to Pradeep, and since we already have the morgan logs, you are right we don't need this. Will remove.